pptx_summarizer.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from pptx_parser import PPTXParser
+from prompt_composer import PromptComposer
+from api_manager import APIManager
+import asyncio
+
+OUTPUT_FILE_NAME = 'result.json'
```

Comment:
One of the requirements was for the json file to have the same name like the original pptx file